analytics.js

```diff
@@ -808,7 +838,11 @@ class Analytics {
           }
         });
       } catch (err) {
-        handleError({ message: `[sendToNative]:${err}` });
+        const destWithErr =
+          succesfulLoadedIntersectClientSuppliedIntegrations[index];
+        handleError({
+          message: `[sendToNative]:${err}. Destination:${destWithErr.name}. Event type:${type}`,
```

Comment:
The event type is already captured in the breadcrumbs of individual APIs. Let's add the destination ID or the actual name set by the user.